lib/vehicle.rb

```diff
@@ -0,0 +1,32 @@
+class Vehicle
+  attr_reader :year, :make, :model, :vehicle, :passengers, :num_adults
+  def initialize(year, make, model, speeding = false, passengers = [])
+    @year = year
+    @make = make
+    @model = model
+    @speeding = speeding
+    @passengers = passengers
+  end
+
+  def speeding?
+    @speeding
+  end
+
+  def speed
+    @speeding = true
+  end
+
+  def add_passenger(passenger)
+    @passengers << passenger
+  end
+
+  def num_adults
+    adults = 0
+    @passengers.each do |passenger|
+      if passenger.adult? == true
```

Comment:
```suggestion
      if passenger.adult?
```

Like the `adult?` method in `Passenger`, this conditional will evaluate as either true or false, so we don't actually have to explicitly ask if it's `== true`.